docs/source/general/guides/drift_guide.rst

```diff
@@ -181,6 +185,32 @@ For code examples, see `here <#tabular-checks>`__
 All of these checks appear also in the `deepchecks interactive demo <https://checks-demo.deepchecks.com>`__, where you can
 insert corruption into the data and see the checks at work.
 
+Text (NLP) Data
+---------------
+
+Regarding `data <#data-drift>`__ or `concept drift <#concept-drift>`__
+In text data, we can't measure drift on the text directly, as text is not structured data that can be measured.
```

Comment:
"that can be measured":
suggesting in replacing this with:
that measures/metrics can directly be calculated on/for.
or
and thus doesn't have inherent/straight-forward ways to calculate these measures.

Reply:
changed to "that can be easily quantified or compared", which seemed more accurate and less repetitive IMO